onnxruntime/core/optimizer/optimizer_execution_frame.cc

```diff
@@ -16,6 +16,19 @@
 
 namespace onnxruntime {
 
+Status OptimizerExecutionFrame::Info::RunSingleKernel(std::vector<int>& fetch_mlvalue_idxs, size_t node_index,
+                                                      std::vector<OrtValue>& fetches, const logging::Logger& logger) {
+  OptimizerExecutionFrame frame(*this, fetch_mlvalue_idxs);
+  auto* kernel = GetKernel(node_index);
+  if (kernel == nullptr) {
+    return Status(ONNXRUNTIME, FAIL);
+  }
```

Comment:
Please always include a reason for the failure to make the error helpful to the user.

Reply:
No. I don't want to put any error message there.